Examples/Swift/FillPatternExample.swift

```diff
@@ -0,0 +1,86 @@
+            CLLocationCoordinate2D(latitude: 38.87189259372385, longitude: -76.7619886599659),
+            CLLocationCoordinate2D(latitude: 38.826521396372705, longitude: -76.7621654264646),
+            CLLocationCoordinate2D(latitude: 38.7821696250172, longitude: -76.77443912189675),
+            CLLocationCoordinate2D(latitude: 38.740771983252394, longitude: -76.7982526104467),
+            CLLocationCoordinate2D(latitude: 38.70413111324496, longitude: -76.83255008385107),
+            CLLocationCoordinate2D(latitude: 38.67383996353737, longitude: -76.87582575693249),
+            CLLocationCoordinate2D(latitude: 38.6512136224523, longitude: -76.92619080795934),
+            CLLocationCoordinate2D(latitude: 38.63723334714361, longitude: -76.98145546133071),
+            CLLocationCoordinate2D(latitude: 38.63250497944332, longitude: -77.03922271728514),
+            CLLocationCoordinate2D(latitude: 38.63723334714361, longitude: -77.09698997323959),
+            CLLocationCoordinate2D(latitude: 38.6512136224523, longitude: -77.15225462661095),
+            CLLocationCoordinate2D(latitude: 38.67383996353737, longitude: -77.20261967763781),
+            CLLocationCoordinate2D(latitude: 38.70413111324496, longitude: -77.24589535071922),
+            CLLocationCoordinate2D(latitude: 38.740771983252394, longitude: -77.2801928241236),
+            CLLocationCoordinate2D(latitude: 38.7821696250172, longitude: -77.30400631267355),
+            CLLocationCoordinate2D(latitude: 38.826521396372705, longitude: -77.3162800081057),
+            CLLocationCoordinate2D(latitude: 38.87189259372385, longitude: -77.3164567746044),
+            CLLocationCoordinate2D(latitude: 38.91630035794387, longitude: -77.30450604902924),
+            CLLocationCoordinate2D(latitude: 38.95780030431052, longitude: -77.28092912463778),
+            CLLocationCoordinate2D(latitude: 38.994572101995104, longitude: -77.24674090531012),
+            CLLocationCoordinate2D(latitude: 39.02500016430364, longitude: -77.20342828294721),
+            CLLocationCoordinate2D(latitude: 39.04774572932767, longitude: -77.15288646542177),
+            CLLocationCoordinate2D(latitude: 39.061806924372846, longitude: -77.09733579147499),
+            CLLocationCoordinate2D(latitude: 39.06656391514849, longitude: -77.03922271728514)
+        ]
+        
+        // Create a shape source based off of a polygon object
+        let shape = MGLPolygon(coordinates: coordinates, count: UInt(coordinates.count))
+        let source = MGLShapeSource(identifier: "circle", shape: shape, options: nil)
+        style.addSource(source)
+        
+        // Create a style layer to be used with the shape source
+        let layer = MGLFillStyleLayer(identifier: "circle", source: source)
+        
+        // Set the fill pattern and opacity for the style layer
+        layer.fillPattern = MGLStyleValue<NSString>(rawValue: "pattern")
```

Comment:
This seems a little magical — perhaps a more specific name than `pattern` and a bit more detailed of an explanation would help.